dash/examples/bench.10.summa/main.cpp

```diff
@@ -335,7 +335,7 @@ void perform_test(
 
   if (myid == 0) {
     if (iteration == 0) {
-      cout << "-- Pattern: " << pattern << endl
+      cout << "-- Pattern: " << dash::internal::typestr(pattern) << endl
```

Comment:
Great, but it would be very useful to have some documentation on the `dash::internal`s